runtime/options.go

```diff
@@ -70,6 +70,8 @@ type CreateOptions struct {
 	Args []string
 	// Environment to configure
 	Env []string
+	// Entrypoint within the folder (e.g. in the case of a mono-repo)
+	Entrypoint string
```

Comment:
What does entrypoint mean? Is it a directory or a main.go? If its main.go let's call this Program, if its a directory let's call it Dir.